src/main/java/com/selenium/course/pages/CommonForm.java

```diff
@@ -0,0 +1,337 @@
+		phoneField.sendKeys(phone);
+	}
+	
+	public void setFax(String fax) {
+		faxField.clear();
+		faxField.sendKeys(fax);
+	}
+	
+	public void setWebSite(String website) {
+		webSiteField.clear();
+		webSiteField.sendKeys(website);
+	}
+
+	public void setTickerSymbol(String tickerSymbol) {
+		tickerSymbolField.clear();
+		tickerSymbolField.sendKeys(tickerSymbol);
+	}
+	
+	public void selectOwnership(String ownership) {
+		selectBox = new Select (ownershipField);
+		selectBox.selectByVisibleText(ownership);
+	}
+	
+	public void setEmployees(String employees) {
+		employeesField.clear();
+		employeesField.sendKeys(employees);
+	}
+	
+	public void setBillingCity(String billingCity) {
+		billingCityField.clear();
+		billingCityField.sendKeys(billingCity);
+	}
+	
+	public void setBillingState(String billingState) {
+		billingStateField.clear();
+		billingStateField.sendKeys(billingState);
+	}
+	
+	public void setBillingZip(String billingZip) {
+		billingZipField.clear();
+		billingZipField.sendKeys(billingZip);
+	}
+	
+	public void setBillingCountry(String billingCountry) {
+		billingCountryField.clear();
+		billingCountryField.sendKeys(billingCountry);
+	}
+	
+	public void setShippingCity(String shippingCity) {
+		shipCityField.clear();
+		shipCityField.sendKeys(shippingCity);
+	}
+	
+	public void setShippingState(String shippingState) {
+		shipStateField.clear();
+		shipStateField.sendKeys(shippingState);
+	}
+	
+	public void setShippingZip(String shippingZip) {
+		shipZipField.clear();
+		shipZipField.sendKeys(shippingZip);
+	}
+	
+	public void setShippingCountry(String shippingCountry) {
+		shipCountryField.clear();
+		shipCountryField.sendKeys(shippingCountry);
+	}
+	
+	public AccountDetail clickSaveAccount() {
+		saveBtn.click();
+		return new AccountDetail();
+	}
+	
+	//Methods to create Leads	
+	public void selectSalutation(String salutation) {
+		selectBox = new Select(salutationOption);
+        selectBox.selectByVisibleText(salutation);
+	}
+	
+	public void setLastName(String lastName) {
+		lastNameField.clear();
+		lastNameField.sendKeys(lastName);
+	}
+	
+	public void setCompany(String companyName) {
+		companyNameField.clear();
+		companyNameField.sendKeys(companyName);
+	}
+
+	public void setCampaing(String campaing) {
+		searchWindow = clickCampaingLookup();
+		searchWindow.findObject(campaing);
+	}
+	
+	private SearchWindow clickCampaingLookup() {
+		findCampaingLookup.click();
+		return new SearchWindow(driver);	
+	}
+	
+	public LeadDetail clickSaveLead() {
+		saveBtn.click();
+		return new LeadDetail();
+	}
+
+	//Methods to create Campaings
+	public void setCampaignName(String name) {
+		campaignNameField.clear();
+		campaignNameField.sendKeys(name);
+	}
+		
+	public void setStatus(boolean status) {
+		WebElement isActiveChBx = isActiveCheckBox;
+		
+		if (status) {
+			if (!isActiveChBx.isSelected()) {
+				isActiveChBx.click();
+			}
+		}
+		else {
+			if (isActiveChBx.isSelected()) {
+				isActiveChBx.click();
+			}
+		}
```

Comment:
I think you can create a generic method to select/clear a checkbox
Create a Commons class for that